example/project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject example/all "MONOLITH"
+(defproject lein-monolith/example.all "MONOLITH"
```

Comment:
I put these in the `lein-monolith` artifact group just in case future tests actually install these projects. That way, they'll automatically get removed from the Maven repository before writing the cache on CircleCI.